adflow/pyADflow.py

```diff
@@ -5250,6 +5250,7 @@ def _getDefaultOptions():
             ],
             "viscWallTreatment": [str, ["constant pressure extrapolation", "linear pressure extrapolation"]],
             "dissipationScalingExponent": [float, 0.67],
+            "acousticScaleFactor": [float, 1.0],
```

Comment:
Do you think adding a warning will be helpful if the user sets the factor to be more than 1.0?

Reply:
I don't think a warning would be very useful. Warnings are easy to ignore, and someone using this option should read the documentation before changing it from the default.